### PR TITLE
Add 41 new PGL assets

### LINKED ISSUE #5167: Add PGL fixture data for all services
Add fixture data to Simorgh for the following PGL assets.

https://www.bbc.com/afaanoromoo/oduu-41217768
https://www.bbc.com/afrique/region-39269126
https://www.bbc.com/amharic/42743191
https://www.bbc.com/arabic/art-and-culture-38260491
https://www.bbc.com/azeri/azerbaijan-44208474
https://www.bbc.com/bengali/news-38827173
https://www.bbc.com/burmese/media-47680015
https://www.bbc.com/gahuza/amakuru-43894701
https://www.bbc.com/gujarati/international-41345658
https://www.bbc.com/hausa/labarai-39326441
https://www.bbc.com/hindi/india-50198153
https://www.bbc.com/igbo/afirika-49666505
https://www.bbc.com/indonesia/indonesia-41635759
https://www.bbc.com/korean/features-41397333
https://www.bbc.com/kyrgyz/world-40847556
https://www.bbc.com/japanese/features-and-analysis-42786589
https://www.bbc.com/marathi/india-42894522
https://www.bbc.com/mundo/deportes-36935058
https://www.bbc.com/nepali/news-50627370
https://www.bbc.com/pashto/arts-and-literature-50230813
https://www.bbc.com/persian/magazine-49281981
https://www.bbc.com/pidgin/tori-49221071
https://www.bbc.com/portuguese/geral-40302633
https://www.bbc.com/punjabi/india-42928885
https://www.bbc.com/russian/features-45782775
https://www.bbc.com/serbian/lat/srbija-46748932
https://www.bbc.com/sinhala/world-37657374
https://www.bbc.com/somali/war-45947544
https://www.bbc.com/swahili/habari-48185450
https://www.bbc.com/tamil/global-47758688
https://www.bbc.com/telugu/india-42321552
https://www.bbc.com/thai/thailand-49950038
https://www.bbc.com/tigrinya/news-49944566
https://www.bbc.com/turkce/haberler-dunya-50924340
https://www.bbc.com/ukchina/trad/cool-britannia-38434549
https://www.bbc.com/ukrainian/features-41278900
https://www.bbc.com/urdu/pakistan-48242478
https://www.bbc.com/uzbek/central-asia-46716844
https://www.bbc.com/vietnamese/world-48605529
https://www.bbc.com/yoruba/media-50970014
https://www.bbc.com/zhongwen/trad/chinese-news-49065935

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #5167 

**Overall change:** 
Add new fixture data for PGLs

**Code changes:**
None
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: data/nepali/cpsAssets/news-50627370.json [hunk 1]
```diff
@@ -0,0 +1,324 @@
+Tree
```
Comment (HarveyPeachey) [inline data/nepali/cpsAssets/news-50627370.json]: Looks like some random text managed to end up here on lines 1-6

Comment (chris-hinds) [reply data/nepali/cpsAssets/news-50627370.json]: oops, good spot

**Review by HarveyPeachey — COMMENTED** [review]
Just a small one

**Review by HarveyPeachey — APPROVED** [review]
LGTM

File: data/nepali/cpsAssets/news-50627370.json (hunk 1 above)
Comment (chris-hinds) [reply data/nepali/cpsAssets/news-50627370.json]: fixed.

**Review by 12 — APPROVED** [review]

**Pull request merged**